flag.go

```diff
@@ -372,17 +372,17 @@ func hasFlag(flags []Flag, fl Flag) bool {
 	return false
 }
 
-func flagFromEnvOrFile(envVars []string, filePath string) (val string, ok bool) {
+func flagFromEnvOrFile(envVars []string, filePath string) (val string, ok bool, source string) {
```

Comment:
Much like returning errors, the "comma ok" idiom typically puts the `ok bool` as the _last_ value of a parameter:

```go
func flagFromEnvOrFile(envVars []string, filePath string) (val, source string, ok bool) {
```

Reply:
Moved it to the end, and added a comment for the function.